report_xlsx_pdf/readme/USAGE.rst

```diff
@@ -0,0 +1,31 @@
+An example of XLSX report for partners on a module called `module_name`:
```

Comment:
This usage comes from report_xlsx it is probably not needed, isn't it?

The real usage is showing how the flag is used, Can you change it?